doc/develop/getting_started/index.rst

```diff
@@ -154,8 +154,8 @@ The current minimum required version for the main dependencies are:
 
          Therefore, we don't recommend using WSL when getting started.
 
-      These instructions must be run in a ``cmd.exe`` command prompt terminal window.
-      In modern version of Windows (10 and later) it is recommended to install the Windows Terminal
+      These instructions must be run in a ``cmd.exe`` Command Prompt window. In modern version
+      of Windows (10 and later) it is recommended to install the Windows Command Prompt
```

Comment:
No, the application in the Microsoft Store is called "Windows Terminal".

Reply:
This section is confusing me then. Are we trying to tell users to install windows terminal for the debugging connection? If so, then let me think of where it should be because it is currently tightly integrated with the command line instructions and it does lead some (at least one) to think they need to go down an odd path.

Reply:
I recommend removal the 2nd sentence entirely to avoid confusion, as cmd.exe (Windows Command Prompt) is present in all supported versions of windows since at least Win2K and cannot be uninstalled.
As @dleach02 has pointed out, the command line instructions in the docs are written for use with cmd.exe.